### PR TITLE
Updates & Polish

### PR DESCRIPTION
**Pull request by deepu105**

### CODE REVIEW

**Comment by deepu105** [comment]
@jdubois I closed the other PR as it had a corrupted commit history. This one has the same changes plz take a look

File: pages/installing_new_languages.md [hunk 1]
```diff
@@ -81,42 +81,18 @@ Here are the steps to install a new language called `new_lang`:
             // jhipster-needle-i18n-language-constant - JHipster will add/remove languages in this array
         ]
 
+    For Angular 2+ add the languege code `new_lang` to the `LANGUAGES` constant defined in `src/main/webapp/app/shared/language/language.constants.ts`
```
Comment (jdubois) [inline pages/installing_new_languages.md]: languege -> language

File: pages/using_angular.md [hunk 2]
```diff
@@ -13,9 +13,18 @@ sitemap:
 
 ## Tooling
 
-Angular 2+ is using TypeScript instead of JavaScript, and as a result some specific tooling is necessary to work efficiently with it.
+Angular 2+ is using TypeScript instead of JavaScript, and as a result some specific tooling is necessary to work efficiently with it. Our [development]({{ site.url }}/development/) workflow for an Angular 2+ application is as below, use `npm` instead of `yarn` if you prefer that.
+
+1. When you generate an application the files are created and at the end of generation `yarn install` task is triggered.
+2. Once `yarn install` is complete it calls the `postInstall` script in `package.json`, this step triggers the `webpack:build` task.
+3. Now you should have all files generated and compiled into the `www` folder inside the `target` or `build` folder based on the build tool(Maven or Gradle) selected.
```
Comment (jdubois) [inline pages/using_angular.md]: There's a space missing in tool(Maven

File: pages/using_angular.md [hunk 3]
```diff
@@ -13,9 +13,18 @@ sitemap:
...
+4. Now run `./mvnw` or `./gradlew` to launch the application server and it should be available at [localhost:8080](localhost:8080) this also serves the client side code compiled from above steps.
```
Comment (jdubois) [inline pages/using_angular.md]: from the above steps

File: pages/using_angular.md [hunk 4]
```diff
@@ -13,9 +13,18 @@ sitemap:
...
+5. Now run `yarn start` in a new terminal to launch Webpack dev-server with BrowserSync. This will take care of compiling your TypeScript code, and automatically reloading your browser.
+
+If you start making changes to the client side code without having `yarn start` running, nothing will be reflected as the changes are not compiled so you need to either run `yarn webpack:build:dev` manually after changes or have `yarn start` running.
+
+You can also force maven/gradle to run webpack task while starting by passing the `webpack` profile like `./mvnw -Pdev,webpack` or `./gradlew -Pdev -Pwebpack`. This is especially helpful after running a `clean` task.
```
Comment (jdubois) [inline pages/using_angular.md]: to run the Webpack task

**Pull request merged**